src/app/main/component/user/components/habit/all-habits/all-habits.component.html

```diff
@@ -23,10 +23,22 @@ <h1 class="main-header">{{ 'user.habit.all-habits.title' | translate }}</h1>
     <div class="habits_header-actions">
       <div class="habits_header-items-found">{{ totalHabits }} {{ 'user.habit.all-habits.found' | translate }}</div>
       <div class="habits-header-buttons">
-        <button class="btn-tiles" (click)="onDisplayModeChange(true)" [ngClass]="{ 'btn-tiles-active': galleryView }" tabindex="0">
+        <button
+          class="btn-tiles"
+          [disabled]="galleryView"
+          [ngClass]="{ 'btn-tiles-active': galleryView }"
+          tabindex="0"
```

Comment:
You don't need `tabindex="0"` for buttons, because they have it by default

Reply:
Forgot to remove, thank you :)